content/Access the world's most powerful retail APIs with Best Buy Ruby - Sandro Damilano.md

```diff
@@ -0,0 +1,43 @@
+## Access the world's most powerful retail APIs with Best Buy Ruby
+
+![Main image](images/blue_led_connection.jpg)
+
+### _What's Best Buy Ruby?_
+
+Best Buy Ruby is a new gem developed entirely here, at Rootstrap. It's basically a wrapper for the different [Best Buy APIs](https://bestbuyapis.github.io/api-documentation/). It offers a simple and intuitive interface that allows users to access the required API with as much customizations as they like.
+
+### _Why use it?_
+
+Best Buy offers a really powerful API. The different options provided range from pagination and sorting methods to incredibly flexible item queries and response formats.
+
+But _with great power, there must also come great responsibility_.
```

Comment:
```suggestion
But _with great power comes great responsibility_.
```

Reply:
I just thought that fancy version was funnier hahaha https://en.wikipedia.org/wiki/With_great_power_comes_great_responsibility
But I sure don't mind changing it!

Reply:
I didn't know that version was also valid, so I'm ok 👌

Reply:
Well, that's actually a very valid point. If that version might not be very well known, let's just go with the common one!